HW_Flask_Login_out/Login.py

```diff
@@ -0,0 +1,69 @@
+"""
+Задание
+
+Создать страницу, на которой будет форма для ввода имени и электронной почты, при отправке которой будет создан
+cookie-файл с данными пользователя, а также будет произведено перенаправление на страницу приветствия, где будет
+отображаться имя пользователя.
+На странице приветствия должна быть кнопка «Выйти», при нажатии на которую будет удалён cookie-файл с данными
+пользователя и произведено перенаправление на страницу ввода имени и электронной почты.
+"""
+
+from flask import (
+    Flask,
+    render_template,
+    redirect,
+    make_response,
+    request,
+    session,
+    url_for,
+)
+
+
+app = Flask(__name__)
+app.secret_key = "secret_key"
+
+
+@app.route("/", methods=["GET", "POST"])
+def index():
+    """
+    Страница с формой ввода
+    пользовательских данных
+    """
+    if request.method == "POST":                             # Если метод запроса POST
+        response = make_response("Cookies")
+        username = request.form.get("username")              # Получаем username и email  из формы
+        user_email = request.form.get("email")
+        if len(user_email) != 0 and len(username) != 0:           # Проверка введены ли данные
```

Comment:
Можно проще, вы ведь метод `get` использовали
```suggestion
        if user_email and username:           # Проверка введены ли данные
```